### PR TITLE
updated README to describe new helper functions

### PR DESCRIPTION
**Pull request by gbarkway**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,4 +12,48 @@ A small wrapper around redis to allow configuration from confit.
...
+--------------------
+
+### Ttl
+Common redis ttl values can be found in `Ttl`.
+```javascript
+req.gb.db.redis.Ttl.oneMinute // => 60
+req.gb.db.redis.Ttl.oneHour // => 3600
+```
+
+### Key Makers
```
Comment (brodneracodes) [inline README.md]: I would call this heading "Redis Keys"

File: README.md [hunk 2]
```diff
@@ -12,4 +12,48 @@ A small wrapper around redis to allow configuration from confit.
...
+
+const mySlowDataFunction = async () => {
+  // slow things here
+};
+
+// return cached value. on cache miss, cache and return result of mySlowdataFunction()
+const myVal = redisClient.memoize(
+  mySlowDataFunction,
+  {
+    key: redisClient.makeKey('redis', 'is', 'cool'), // defaults to name of function
```
Comment (brodneracodes) [inline README.md:56]: I think I would first provide an example without the key, as that will be the common use case. Then show that you can optionally specify the key.

**Review by brodneracodes — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (gbarkway) [reply README.md:56]: I think I disagree that using the default key is common usage.  In the three services we're calling this function from, we're specifying a key every time.

Comment (brodneracodes) [reply README.md:56]: That' true. I forgot about the case where the function is parameterized. No this is good then.

**Review by brodneracodes — APPROVED** [review]

**Pull request merged**